neuro-sdk/src/neuro_sdk/_jobs.py

```diff
@@ -352,10 +353,14 @@ async def run(
         life_span: Optional[float] = None,
         org_name: Union[Optional[str], OrgNameSentinel] = ORG_NAME_SENTINEL,
         priority: Optional[JobPriority] = None,
+        project_name: Optional[str] = None,
     ) -> JobDescription:
         url = self._config.api_url / "jobs"
+        if not project_name:
+            project_name = self._config.project_name or self._config.username
```

Comment:
Yes, API wouldn't allow users to create jobs not in their projects or w.o. projects at all, we are safe on the backend side.

As for SDK/CLI, we could either make the `project_name` parameter of `Jobs.run` and `Jobs.start` obligatory, OR rely on `self._config.project_name` and raise an exception at `_job_to_api` stating smth around "Project is not specified, make sure you have created one, or reload configuration before running the jobs". I'm more for 2nd approach, wdyt?

Reply:
From a user experience point of view, 2 is preferred. Most users rarely switch projects, and having to constantly retype even though it rarely changes would be a hassle.

Reply:
This is similar in a way to clusters - most users will never access more than one

Reply:
these all seem valid points. by only concern is us explicitly using `self._config.username`. by the time this code executes all existing users will have their default project created and even pulled from `/config`. otherwise asking to create a project seems fine.

Reply:
in 890658ad7a11e12e4a0eb8c3b2a2a84f0ff24c11 I've added one extra propery of `config` named `project_name_or_raise` which raises exception `The current project is not selected. Please create one with 'neuro admin add-project', or switch to the existing one with 'neuro config switch-project'.` So this property to start a job if no `project` parameter is specified.
 wdyt, @dalazx ?